### DISCUSSION TITLE
Routes over private roads.

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [gis-ops/valhalla-app#201] [Q&A]
Q: Why the demo server is routing over private roads.  
https://www.openstreetmap.org/way/538364148
(Shortest)  Is there a specific setting to prevent this ? 

https://valhalla.openstreetmap.de/directions?profile=pedestrian&wps=4.002585411071778%2C51.05098550341861%2C4.0058255195617685%2C51.055463603728086

**Comment by nilsnolde** [comment]
If it’s the only way to get to a place it will. Otherwise it shouldn’t. Anyways, this is a question for https://github.com/valhalla/valhalla

**Reply by nilsnolde** [reply]
And I’d recommend providing a _minimal_ example, plus the a link to the OSM way.
